Marlin/Marlin_main.cpp

```diff
@@ -347,20 +347,31 @@ bool target_direction;
 #endif
 
 #ifdef DELTA
+
+  #define TOWER_1 X_AXIS
+  #define TOWER_2 Y_AXIS
+  #define TOWER_3 Z_AXIS
```

Comment:
I just did a `COREXZ` to match (I hope) the way `COREXY` works, and in that process I just stuck with `A_AXIS` and `B_AXIS`, and I added `C_AXIS` because it wasn't created yet. A lot of reference material just labels the delta towers with `XYZ` but Marlin is moving towards `ABC` when referring directly to steppers. So for forward-compatibility, it would be ok to go with `A_AXIS`, `B_AXIS`, and `C_AXIS` below.